assets/templates/signup.html

```diff
@@ -40,54 +40,62 @@ <h5>Regístrate</h5>
       <p class="content-subtitle">Crea una cuenta GRATIS hasta 500 Suscriptores.</p><!-- Title -->
       <form class="signup-form">
         <fieldset>
-          <legend>Creación de cuenta</legend>
-          <ul>
-            <li class="holder-small">
-              <label for="name">Nombre:</label>
-              <input type="name" name="lastname" placeholder="Nombre">
-            </li>
-            <li class="holder-small">
-              <label for="lastname">Apellido:</label>
-              <input type="lastname" name="name" placeholder="Apellido">
-            </li>
+          <legend>Datos personales</legend>
+          <ul class="holder-group">
             <li>
+              <ul class="holder-group">
+                <li class="holder-item holder-item--small">
+                  <label for="name">Nombre:</label>
+                  <input type="name" name="name" id="name" placeholder="Nombre">
+                <li class="holder-item holder-item--small">
+                  <label for="lastname">Apellido:</label>
+                  <input type="lastname" name="name" id="lastname" placeholder="Apellido">
+                </li>
+              </ul>
+            </li>
+            <li class="holder-item holder-item--big">
```

Comment:
i'm not sure but, how about having a "holder-item" main class and have "big" and "small" modifiers classes to only adjust the size?